jsparagus/ordered.py

```diff
@@ -94,6 +94,12 @@ def __isub__(self, other: AbstractSet[T]) -> OrderedSet[T]:
                 self.remove(v)
         return self
 
+    def is_disjoint(self, other: AbstractSet[T]) -> bool:
```

Comment:
I hate this, but the official spelling is: https://docs.python.org/3.8/library/stdtypes.html#frozenset.isdisjoint

Don't change it if you don't want to.

Reply:
I tried to name it `isdisjoint`, but our static-checker complained about it.